package.json

```diff
@@ -56,6 +59,9 @@
     "plugins": [
       "@typescript-eslint"
     ],
+    "globals": {
+      "ReCaptchaV2": "readonly"
```

Comment:
ah, so declaring here allows for [@types/grecaptcha](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/grecaptcha/grecaptcha-tests.ts) to be accessible without import. - took a bit of time to figure that out, was scouring for an `import` 😅